pkg/aws/utils.go

```diff
@@ -151,13 +151,13 @@ func GetAWSClientForUserRegion(reporter *rprtr.Object, logger *logrus.Logger) Cl
 // Validations will validate if CF stack/users exist
 func CheckStackReadyForCreateCluster(reporter *rprtr.Object, logger *logrus.Logger) {
 	client := GetAWSClientForUserRegion(reporter, logger)
-	reporter.Debugf("Validating cloudformation stack exists")
-	stackExist, _, err := client.CheckStackReadyOrNotExisting(OsdCcsAdminStackName)
-	if !stackExist || err != nil {
-		reporter.Errorf("Cloudformation stack does not exist. Run `rosa init` first")
+	reporter.Debugf("Validating admin user exists")
+	err := client.CheckAdminUserNotExisting(AdminUserName)
+	if err == nil {
+		reporter.Errorf("IAM user '%s' does not exist. Run `rosa init` first", AdminUserName)
```

Comment:
I think you should leave this function as is (unless we don't use it elsewhere) and create a new function `CheckAdminUserExists` that calls `resp, err := c.iamClient.GetUser(&iam.GetUserInput{UserName: aws.String(AdminUserName)})`. Otherwise we're relying on the existence of an error to determine that everything is OK.